### PR TITLE
Added a script to process the FB data to have the right format

### PR DESCRIPTION
**Pull request by woniesong92**
This PR is a starting point for the following:

https://github.com/hunkim/deep-facebook-commenter/issues/3

### CODE REVIEW

**Comment by woniesong92** [comment]
It uses either tf.txt or dcoding.txt as input and makes an output with the following format:

p c1
p c2
...
p cn

It doesn't take the number of likes into account. We could try other variations..

/cc @hunkim

File: scripts/prepare_fb_data.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  raw = line.split("\t")
+  post = {'num_likes': raw[0], 'body': raw[1].rstrip()}
+  raw_comments = raw[2:]
+  comments = []
+  for i in range(0, len(raw_comments), 2):
+    comments.append({'num_likes': raw_comments[i], 'body': raw_comments[i+1].rstrip()})
+  return {'post': post, 'comments': comments}
+
+def process_raw_data(filename, output):
+  with open(filename) as f1:
```
Comment (hunkim) [inline scripts/prepare_fb_data.py]: Thanks! Could we use gzopen to save disk space? Also, we need encoding, 'utf-8' somewhere. 

See https://github.com/hunkimForks/DeepQA/blob/master/chatbot/fbdata.py

**Comment by hunkim** [comment]
Wonderful. It's a good start.

Later, could we directly transform to the data format for chatbot-retrieval or add one helper method for chatbot-retrieval to read fbdata/*.gz directly.

**Comment by hunkim** [comment]
Cool!

**Pull request merged**